### DISCUSSION TITLE
Is it intentional that wslu is not installed by default with Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by vbrozik** [wslutilities/wslu#234] [Q&A] [answered]
**Background Information:**
 - Build: any
 - WSL: Ubuntu 22.04

**Describe the bug**
1. `wslu` is not installed any more by default with Ubuntu 22.04 for WSL.
2. The documentation does not mention the change: https://wslutiliti.es/wslu/install.html#ubuntu

**To Reproduce**
* Install WSL with Ubuntu 22.04 and check the presence of the package:
```
$ dpkg -l wslu
dpkg-query: no packages found matching wslu
```
* Check the dependencies of the package `ubuntu-wsl` which caused installation of `wslu` in older versions.
https://packages.ubuntu.com/jammy/ubuntu-wsl

**Expected behavior**
Such a useful package for WSL like `wslu` should still be installed by default.

**Additional context**
Changelog of `ubuntu-wsl` with the package removal:
http://changelogs.ubuntu.com/changelogs/pool/main/u/ubuntu-meta/ubuntu-meta_1.481/changelog

**Comment by vbrozik** [comment]
Finally I was able to find the commit(s). The removal of `wslu` was done in two steps - for Ubuntu 21.10 and 22.04:

* `ubuntu-meta` 1.470, 2021-06-08 for Ubuntu 21.10 - removal for targets `armhf` `ppc64el` `riscv64` `s390x`
  * https://git.launchpad.net/ubuntu/+source/ubuntu-meta/commit/?h=ubuntu/jammy&id=e84f8358d8e78cb4a38fe65dcaf5d27808dd7c4e
* `ubuntu-meta` 1.474, 2022-01-29 for Ubuntu 22.04 - removal for targets **`amd64`** `arm64`
  * https://git.launchpad.net/ubuntu/+source/ubuntu-meta/commit/?h=ubuntu/jammy&id=c846724c7d249e0ff6d3a38a6a91468794e910eb

Unfortunately I still have no idea how such changes are decided on in Ubuntu. Searching for `wsl` or `wslu` in the bug-tracker returns no results: https://bugs.launchpad.net/ubuntu/+source/ubuntu-meta Do you know where I can follow the decision process of changes in Ubuntu?

I have asked a question about the decision process on Ask Ubuntu: [How to find a reason for a change in Ubuntu? (example: removal of package wslu from the default installation)](https://askubuntu.com/q/1409323/67132)

**Comment by car-meows** [comment]
Since I have left Canonical last year, currently it's Canonical's decision on changing the meta packages and I have no control over it. If you want to install for 22.04, you can install it via PPA right now.

**Reply by ggirard07** [reply]
New user here... I think documentation should be update at least as it is quite misleading for new user, reading it is installed by default while actually it is not.

**Reply by car-meows** [reply]
> New user here... I think documentation should be update at least as it is quite misleading for new user, reading it is installed by default while actually it is not.

You are right... I am updating the document now

**Comment by vbrozik** [comment, marked as answer]
Thank you, I see that the documentation is fixed now.
https://wslutiliti.es/wslu/install.html#ubuntu
> Starting from Ubuntu 22.04 LTS, Ubuntu on WSL no longer bundled with Ubuntu on WSL.
